plugin.video.catchuptvandmore/resources/lib/channels/be/rtlplaybe.py

```diff
@@ -236,6 +236,7 @@ def list_videos_search(plugin, search_query, item_id, page, **kwargs):
                         item_post_treatment(item,
                                             is_playable=True,
                                             is_downloadable=is_downloadable)
+                        yield
```

Comment:
this looks like a leftover? You yield the actual item below

Reply:
good catch. I fixed that